composer.json

```diff
@@ -15,20 +15,15 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
-        "silverstripe/framework": "~3.1",
-        "symbiote/silverstripe-queuedjobs": "^3",
-        "zendframework/zend-ldap": "^2.5.1",
-        "zendframework/zend-authentication": "^2.5.1",
-        "zendframework/zend-session": "^2.5.1",
-        "onelogin/php-saml": "^2.10.7"
+        "php": "^8.1",
+        "silverstripe/framework": "^5",
+        "symbiote/silverstripe-queuedjobs": "^4",
```

Comment:
```suggestion
        "symbiote/silverstripe-queuedjobs": "^5",
```

Reply:
Actually - the `4` version of this module only supports framework 3 - we probably don't want a CMS 5 version - check with @maxime-rainville